fem/libceed/ceed.hpp

```diff
@@ -91,17 +91,38 @@ struct CeedData
 
 };
 
+/** This structure contains the data to assemble a PA operator with libCEED.
+ *  See libceed/mass.cpp or libceed/diffusion.cpp for examples. */
```

Comment:
(minor) The `*` at the beginning of the line in the comment block is not consistent with the rest of the Doxygen comments in MFEM (also appears several other places in this file).

Reply:
I think I "fixed" this before merging... did I miss an instance?

Reply:
No, I just followed some github notification blindly to an outdated diff. Everything is fine, never mind.

Reply:
Ok. I agree with your point though -- we should try to unify the style :)

Reply:
Sorry vscode is adding these automatically... I'll be more careful.